src/net_processing.cpp

```diff
@@ -4375,8 +4375,6 @@ void PeerManagerImpl::MaybeSendAddr(CNode& node, std::chrono::microseconds curre
 
     assert(node.m_addr_known);
 
-    const CNetMsgMaker msgMaker(node.GetCommonVersion());
-
     // Periodically advertise our local address to the peer.
     if (fListen && !::ChainstateActive().IsInitialBlockDownload() &&
```

Comment:
might be worth updating to `m_chainman.ActivateChainstate()`? 

h/t https://github.com/bitcoin/bitcoin/pull/21327#discussion_r585433607

Reply:
This isn't adding a new `::ChainstateActive()` like #21327 was. I think changing this call to be `m_chainman.ActiveChainstate()` would be adding more noise/distraction for reviewers.

Reply:
ok. since this is a refactor PR anyway, as a reviewer I would not mind, but its no worries :)